app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -302,7 +304,7 @@ class ApiServiceTest {
 
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.getUserById(wrongUserId)
+        val actualResponse = apiService.getUserById(bearerToken, wrongUserId)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver